pkg/references/conversions_test.go

```diff
@@ -0,0 +1,401 @@
+package references
```

Comment:
Nice test coverage. We should add coverage reports to PRs.

Reply:
:D sure, will do

Reply:
Also track as an issue? :smiley: